contracts/V1/DisputeTypes.sol

```diff
@@ -182,19 +179,20 @@ struct DisputeAuditingData {
     StateSnapshot[] milestoneSnapshots; //for K milestones there will be K-1 snapshots, since the first milestone is the genesisSnapshot
     bytes latestStateStateMachineState;
     JoinChannelBlock[] joinChannelBlocks;
+    // ========================== optional ===============================
     Dispute previousDispute; // (optional) needed to verify 'this' dispute genesis against the previous dispute outputSnapshot or genesisSnapshot (in the case of a recursive dispute) - if not present, genesis is the latest on-chain Snapshot
     uint previousDisputeTimestamp; // (optional) needed to verify the commitment of the previous dispute
 }
 
-struct DisputeData{
+struct DisputeData {
     DisputePair[] disputePairs;
-    address[] onChainSlashedParticipants;
+    address[] slashedParticipants;
```

Comment:
In my opinion this (onChainSlashes) should remain as a clear indicator that it's different from slashedParticipants, since populating this actually means that auditing was executed on-chain, while if not executed they would still be slashed 'off-chain' and committed 'on-chain' but this explicit storage array wouldn't reflect it and only when the snapshot is eventually updated would the participant set reflect it.

Since every time we have a fraud proof we actually run auditing on-chain to actually populate this array and prevent the byzantine participant from participating in the 'dispute game' going forward, this is logically correct even if we keep the obfuscated 'slashedParticipants'